src/clients/TSCClient.js

```diff
@@ -13,6 +13,7 @@ export class TSCClient extends ClientBase {
       const [match, path, lineNumberString, message] =
         line.match(/(.*\.tsx?)[\(:](.*)[:,].*(error.*)/) || []
       if (!match) {
+        if (acc.length === 0) return acc // there could be output that isn't related - we will skip that
```

Comment:
If I'm reading this correctly, what we really want is to skip any output until we get to the first line that matches that regex.

What would you think about putting that _before_ the reduce loop ever starts? We may have a chance, in that way, to communicate the intent more clearly without a comment.

I'm thinking something like

```js
const ERROR_REGEX = /(.*\.tsx?)[\(:](.*)[:,].*(error.*)/

const firstLineWeCareAbout = this.output.findIndex(line => line.match(ERROR_REGEX))
const linesWeCareAbout = this.output.slice(firstLineWeCareAbout)

return (this._annotations = linesWeCareAbout.reduce((acc, line) => {
```